README.rst

```diff
@@ -114,17 +103,9 @@ Documentation
 Installation
 ============
 
-The packages you have to install depend on the features you want to use. The
-H-SAF image products are disseminated in BUFR (H07, H16, H103, H101, H102, H08)
-or GRIB (H14) format. So to read them you will have to install the appropriate
-packages which will be explained shortly. Unfortunately neither BUFR nor GRIB
-readers work on Windows so if you need these formats then Linux or OS X are your
-only options.
+The packages you have to install depend on the features you want to use. The H SAF soil moisture NRT products are disseminated in BUFR (H16, H103, H101, H102, H08) or GRIB (H14) format. So to read them you will have to install the appropriate packages which will be explained shortly. Unfortunately neither BUFR nor GRIB readers work on Windows so if you need these formats then Linux or OS X are your only options.
```

Comment:
I usually have a paragraph in one line and the editor takes care of line breaks. Should we break text at 80?